/**:
  ros__parameters:
    device_name: /dev/ttyUSB0
    baud_rate: 115200
    flow_control: none
    parity: none
    stop_bits: "1"
    broadcast_tf: false
    linear_acceleration_stddev: 0.0039228
    angular_velocity_stddev: 0.00087266
    pitch_roll_stddev: 0.017453
    yaw_stddev: 0.087266